components/Clock.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="clock">
-    <span>{{ currentDate }} | {{ myTimer() }}</span>
-    <!-- <span id="txt">{{ startTime() }}</span> -->
+    <!-- <span>{{ currentDate }} | {{ myTimer() }}</span> -->
+    <span id="txt">{{ timer }}{{ startTime() }}</span>
```

Comment:
@rajakvk Here we are using `timer` as well as `startTime()` function. Is it feasible?
Can't we use either of them only?
What do you think?